contrib/IECoreArnold/src/IECoreArnold/outputDriver/OutputDriver.cpp

```diff
@@ -134,6 +137,12 @@ void driverOpen( AtNode *node, struct AtOutputIterator *iterator, AtBBox2 displa
 			namePrefix = std::string( name ) + ".";
 		}
 
+		const StringData *label = parameters->member< StringData >( "label" );
```

Comment:
I think Dan was hoping that 'beauty' could be considered a standard, on the basis the Gaffer Outputs node comes with 'beauty' outputs by default (albeit via gui config).